vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -387,6 +387,13 @@ func RunGet(f *cmdutil.Factory, out io.Writer, errOut io.Writer, cmd *cobra.Comm
 				allErrs = append(allErrs, err)
 				continue
 			}
+
+			// add linebreak between resource groups (if there is more than one)
+			// skip linebreak above first resource group
+			if lastMapping != nil {
+				fmt.Fprintf(out, "%s\n", "")
```

Comment:
Thanks for the feedback, it now prints linebreaks to stderr and honors the `--no-headers` flag